Gemfile

```diff
@@ -9,6 +9,7 @@ gem "optimist"
 
 gem "sources-api-client",         :git => "https://github.com/ManageIQ/sources-api-client-ruby", :branch => "master"
 gem "topological_inventory-core", :git => "https://github.com/ManageIQ/topological_inventory-core", :branch => "master"
+gem "inventory_refresh",          :git => "https://github.com/ManageIQ/inventory_refresh", :branch => "master"
```

Comment:
We should add this to the `-core` gemspec instead of here since it is included by the Schema class

Reply:
right, it's in core's Gemfile, that's why it don't work now. I'll create PR

Reply:
Ah yeah because we haven't pushed releases for the current master branch.  It has stabilized a lot so I'll push a v0.3.0 release.